spec/spec_helper.rb

```diff
@@ -48,6 +48,18 @@
   c.after(:suite) do
     RSpec::Puppet::Coverage.report!(0)
   end
+
+  # Filter backtrace noise
+  backtrace_exclusion_patterns = [
+    %r{spec_helper},
+    %r{gems},
+  ]
+
+  if c.respond_to?(:backtrace_exclusion_patterns)
+    c.backtrace_exclusion_patterns = backtrace_exclusion_patterns
+  elsif c.respond_to?(:backtrace_clean_patterns)
+    c.backtrace_clean_patterns = backtrace_exclusion_patterns
+  end
```

Comment:
Shouldn't this be contributed to https://github.com/puppetlabs/puppetlabs_spec_helper?

Reply:
Hi @ekohl, 
For now we will keep these changes in the pdk-templates and if this works out nicely for the currently supported modules then we will probably port it to the puppetlabs_spec_helper and this change will be removed on a future pdk update.